spring-cloud-dataflow-docs/pom.xml

```diff
@@ -12,7 +12,7 @@
 	<description>Spring Cloud Data Flow Docs</description>
 	<properties>
 		<main.basedir>${basedir}/..</main.basedir>
-		<docs.resources.version>0.2.1.RELEASE</docs.resources.version>
+		<docs.resources.version>0.2.5</docs.resources.version>
```

Comment:
**[UNRELATED]** `0.2.1.RELEASE` not available in snapshots nor central, `0.2.5` is what we need to be using which is in fact on central.